lib/vsc/utils/fancylogger.py

```diff
@@ -215,11 +227,11 @@ class MissingLevelName(KeyError):
 
 def getLevelInt(level_name):
     """Given a level name, return the int value"""
-    if not isinstance(level_name, basestring):
+    if not isinstance(level_name, str):
```

Comment:
euhm, that is not ok, right? also no point in hiding the "lack of basestring" mess in python3, make a py23 module and fix this

Reply:
fixed using `not is_string(level_name)`, where `is_string` is provided by `vsc.utils.py2vs3`